ui/src/pages/session-details/timeline-slider/styles.module.scss

```
@@ -5,9 +5,9 @@
 
   .label {
     display: block;
-    @include body-small();
+    @include body-overline();
```

Stylelint (17.12.0)

[error] Unexpected parentheses in argumentless mixin "body-overline" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)